docs/platforms/javascript/common/tracing/configure-sampling/index.mdx

```diff
@@ -96,98 +119,16 @@ tracesSampler: (samplingContext) => {
 }
 ```
 
-## The Sampling Context Object
-
-When the `tracesSampler` function is called, it receives a `samplingContext` object with valuable information to help make sampling decisions:
-
-```typescript
-typescriptCopyinterface SamplingContext {
-  // Name of the span/transaction
-  name: string;
-  
-  // Initial attributes of the span/transaction
-  attributes: SpanAttributes | undefined;
-  
-  // Whether the parent span was sampled (undefined if no incoming trace)
-  parentSampled: boolean | undefined;
-  
-  // Sample rate from incoming trace (undefined if no incoming trace)
-  parentSampleRate: number | undefined;
-  
-  // Utility function to inherit parent decision or fallback
-  inheritOrSampleWith: (fallbackRate: number) => number;
-}
-```
-
-The sampling context contains:
-
-- `name`: The name of the transaction/span
-- `attributes`: Initial tags and attributes set on the transaction
-- `parentSampled`: Whether the parent transaction was sampled (for distributed tracing)
-- `parentSampleRate`: The sample rate used in the parent transaction
-- `inheritOrSampleWith`: A utility function to handle inheritance logic (recommended)
-
-## Inheritance in Distributed Tracing
-
-In distributed systems, trace information is propagated between services. The inheritOrSampleWith function simplifies handling parent sampling decisions:
-
-```javascript
-tracesSampler: (samplingContext) => {
-  const { name, inheritOrSampleWith } = samplingContext;
-  
-  // Apply specific rules first
-  if (name.includes('critical-path')) {
-    return 1.0; // Always sample
-  }
-  
-  // Otherwise inherit parent sampling decision or fall back to 0.1
-  return inheritOrSampleWith(0.1);
-}
-```
-This approach ensures consistent sampling decisions across your entire distributed trace. All transactions in a given trace will share the same sampling decision, preventing broken or incomplete traces.
-
-**Note:** The `inheritOrSampleWith` helper was introduced in version 9 of the SDK. For earlier versions, you can implement similar logic manually using the `parentSampled` property.
-
 ## Sampling Decision Precedence
 
 When multiple sampling mechanisms could apply, Sentry follows this order of precedence:
 
-- If `tracesSampler` is defined, its decision is used (can consider parent sampling)
-- If no `tracesSampler` but parent sampling is available, parent decision is used
+- If a sampling decision is passed to `startSpan`, that decision is used
```

Comment:
```suggestion
```

this is not correct anymore for JS, you cannot pass a sampling decision like this!